packages/okta-react/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "React support for Okta",
   "main": "./dist/index.js",
   "scripts": {
-    "build": "rimraf dist/ && yarn build:package-info && babel src -d dist",
+    "build": "rimraf dist/ && yarn build:package-info && babel src -d dist -s",
```

Comment:
Why not set this in the .babelrc?

Reply:
Looks like babelrc sourceMaps option only works for `inline`
https://github.com/babel/babel/issues/2484#issuecomment-266607686